### PR TITLE
Week 10 - Project Survey - Johanna Rexin

### PR DESCRIPTION
**Pull request by johannarexin**
Hey,
Here is my week 10 Project Survey.

https://johannarexin-survey.netlify.app/

Best, Johanna

### CODE REVIEW

**Review by liza-minican — COMMENTED** [review]
Johanna, well done!!  I liked your survey and how you structured your code.
Let me suggest you a piece of code which could help to reach this requirement for the project. - There should be a submit button. When pressed your app should hide the input form and show a summary of the user's submissions.
You already have the handleSubmit function in your code.
` {! showSummary ? ( 
          <form onSubmit={handleSubmit}>
/*here you can have your form*/` 
`  </form>
  ) : ( <Summary   />    /*here you can have your summary with props*/
      )
}`

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,103 @@
-import React from 'react'
+import React, { useState } from 'react'
+import "./index.css"
```
Comment (EmOlsz) [inline code/src/App.js:2]: There is a convention which says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, `style.css` should be separated as well and located below local dependencies. Ideally, it should look like this :

```jsx
import React from 'react';
import ReactDOM from 'react-dom';

import MyComponent1 from './components/MyComponent1';
import MyComponent2 from './components/MyComponent2';
import MyComponent3 from './components/MyComponent3';

import './style.css';
```

So, final order is :

1. Outer dependencies
2. Local dependencies
3. Style file(s)

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,103 @@
...
+      </label>
+
+      <label ClassName="option">
+        Coffee      
+        <input
+          type="radio"
+          checked={drink === "coffee"}
+          value="coffee"
+          name="drink"
+          onChange={(e) => {setDrink (e.target.value)}}
```
Comment (EmOlsz) [inline code/src/App.js:76]: we don't need extra curly brackets around. This is completely enough :

```
onChange={e => setDrink(e.target.value)}
```

**Review by EmOlsz — COMMENTED** [review]
Hey Johanna! According to Liza's review, let's remove `br` tags and change them with proper CSS styles. After that, I'll approve your project :)

**Comment by johannarexin** [comment]
Hi Maks, Now I've done the changes and I also made the site responsive :) Bets, Johanna

**Comment by EmOlsz** [comment]
Great job, approved :)

**Pull request closed without merging**